src/frontend/scss/settings/_fonts.scss

```diff
@@ -1,8 +1,48 @@
-//
-// https://fonts.google.com/?query=Montserrat&selection.family=Hind:wght@400;600|Montserrat:wght@300;700;800&sidebar.open
```

Comment:
You. Hanged the weight? Same question for hind?

Reply:
Yes, I do not import all font variants only those we were using previously.

**Hind**:
400: regular
600: semibold

**Montserrat**:
300: light
500: regular
700: bold
800: extrabold

Reply:
👍